tools/notification-configuration/github-codeowner-subscriber/Program.cs

```diff
@@ -52,22 +47,19 @@ class Program
             }))
 #pragma warning restore CS0618 // Type or member is obsolete
             {
-                var devOpsService = AzureDevOpsService.CreateAzureDevOpsService(
-                    Environment.GetEnvironmentVariable(devOpsTokenVar),
+                var devOpsService = AzureDevOpsService.CreateAzureDevOpsService(devOpsTokenVar,
                     $"https://dev.azure.com/{organization}/",
                     loggerFactory.CreateLogger<AzureDevOpsService>()
                 );
                 
                 var gitHubServiceLogger = loggerFactory.CreateLogger<GitHubService>();
                 var gitHubService = new GitHubService(gitHubServiceLogger);
-
-                var githubNameResolver = new GitHubNameResolver(
-                    Environment.GetEnvironmentVariable(aadAppIdVar),
-                    Environment.GetEnvironmentVariable(aadAppSecretVar),
-                    Environment.GetEnvironmentVariable(aadTenantVar),
-                    Environment.GetEnvironmentVariable(kustoUrlVar),
-                    Environment.GetEnvironmentVariable(kustoDatabaseVar),
-                    Environment.GetEnvironmentVariable(kustoTableVar),
+                var options = new AadClientSecretOptions();
```

Comment:
Why create the options class as opposed to just creating the credential object?

Reply:
Use the credential object instead